packages/math-input/package.json

```diff
@@ -56,29 +56,29 @@
         "katex": "0.11.1",
         "perseus-build-settings": "^0.4.1",
         "prop-types": "15.6.1",
-        "react": ">= 18",
-        "react-dom": ">= 18",
-        "react-router": "^5.2.1",
-        "react-router-dom": "^5.3.0",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-router": "5.3.4",
+        "react-router-dom": "5.3.4",
```

Comment:
**thought**: I wonder if these are even used/needed by `math-input`?  We don't import them anywhere. I wonder if this is just cruft from when we moved Perseus back into this repo. No change necessary, just musing.